specification/keyvault/data-plane/Security.KeyVault.SecurityDomain/tspconfig.yaml

```diff
@@ -0,0 +1,30 @@
+parameters:
+  "service-dir":
+    default: "sdk/keyvault"
+
+emit:
+  - "@azure-tools/typespec-autorest"
+
+linter:
+  extends:
+    - "@azure-tools/typespec-azure-rulesets/data-plane"
+
+options:
+  "@azure-tools/typespec-autorest":
+    azure-resource-provider-folder: "data-plane"
+    emitter-output-dir: "{project-root}/.."
+    output-file: "{azure-resource-provider-folder}/{service-name}/{version-status}/{version}/securitydomain-sdk.json"
+  # Uncomment this line and add "@azure-tools/typespec-python" to your package.json to generate Python code
+  "@azure-tools/typespec-python":
```

Comment:
Doesn't this need to be up in the `emit` array as well, or is that only for the python SDK repo?